sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/PodInfoBuilder.java

```diff
@@ -42,23 +42,28 @@
     private Protos.ExecutorInfo.Builder executorBuilder;
     private final PodInstance podInstance;
     private final Map<String, Map<String, String>> lastTaskEnvs;
+    private final boolean useDefaultExecutor;
 
     public PodInfoBuilder(
             PodInstanceRequirement podInstanceRequirement,
             String serviceName,
             UUID targetConfigId,
             SchedulerFlags schedulerFlags,
-            Collection<Protos.TaskInfo> currentPodTasks)
-                    throws InvalidRequirementException {
+            Collection<Protos.TaskInfo> currentPodTasks,
+            Protos.FrameworkID frameworkID,
+            boolean useDefaultExecutor) throws InvalidRequirementException {
```

Comment:
IMO it'll be a little easier to read if this parameter is defined as an enum rather than a bool:

```
public enum ExecutorType {
  DEFAULT,
  CUSTOM
}
```

Then callers would look like this:
```
PodInfoBuilder b = new PodInfoBuilder(..., ExecutorType.DEFAULT);
```